{{cookiecutter.app_name}}/README.md

```diff
@@ -62,9 +62,9 @@ Once you have installed your DBMS, run the following to create your app's
 database tables and perform the initial migration
 
 ```bash
-flask db init
-flask db migrate
-flask db upgrade
+python -m flask db init
+python -m flask db migrate
+python -m flask db upgrade
```

Comment:
Assuming the user has followed the installation instructions above (_i.e._ either running `python -m pipenv install --dev` or `pip install -r requirements/dev.txt`), `flask` should be added into the `/bin` of the appropriate virtualenv. Therefore, running these commands should use the correct version of flask and the `python -m` is unnecessary. Have you found this to not be the case?